src/modules/thumb/Thumb.js

```diff
@@ -0,0 +1,82 @@
+import React, { useEffect, useState } from "react";
+import { node, string } from "prop-types";
+import { useImage } from 'react-image'
+import { useClassNames } from "src/hooks/useClassNames";
+import Icon from "lib/Icon";
+import bolt from "../../../assets/icons/bolt.svg";
+
+function Thumb({ src, altText, title, fallback, children, ...props }) {
```

Comment:
`Thumb` is pretty cool out the box it;

- handles the loading and failures of showing a thumbnail
- it'll wait a minimal amount of time before showing the image to reduce UI movement and weird whitespace UI issues that you can get with caching and things actually rendering in the browser
- For now the default is fallback pants but this will be updated very soon and in the consuming app it'll also be set anyways